lxd/main.go

```diff
@@ -15,6 +15,7 @@ import (
 
 // Global arguments
 var argAuto = gnuflag.Bool("auto", false, "")
+var argPreseed = gnuflag.Bool("preseed", false, "")
```

Comment:
@stgraber do you want to change the behavior in this branch then? Because if we plan to make the filename mandatory , then we can't "formally" do that after merging this branch since it would add a backward incompatibility. If you prefer the semantics suggested by @albertodonato , plus making the filename mandatory, I'd rather implement it in this branch.